clouddriver-kubernetes/src/test/java/com/netflix/spinnaker/clouddriver/kubernetes/op/KubernetesDeleteManifestOperationTest.java

```diff
@@ -96,7 +96,7 @@ void setupTest() {
   }
 
   @Test
-  public void orphanDependantsTrueTest() throws IOException {
+  public void orphanDependentsTrueTest() throws IOException {
```

Comment:
nit: ideally the initial commit wouldn't have the typos.

Reply:
Fixed typo in first commit

Reply:
Thanks.